### PR TITLE
set the default state prep label to a psi ket

### PR DESCRIPTION
**Pull request by timmysilv**
**Context:**
`QubitStateVector` shrinks down reaaaally small when drawing, and it doesn't look great.

**Description of the Change:**
All operators that inherit from StatePrep will have a label of $\vert \psi \rangle$ by default

**Benefits:**
Our drawings are much nicer

**Possible Drawbacks:**
You can't tell the difference between StatePrep operations in drawings

**Related GitHub Issues:**
Found while working on PennyLaneAI/qml#812

An example:
image

and the text drawer:

```
0: ─╭|ψ⟩──RX(0.10)─╭●─╭X─┤  Probs
1: ─├|ψ⟩───────────╰X─│──┤  Probs
2: ─╰|ψ⟩──RY(0.20)────╰●─┤  Probs
```

### CODE REVIEW

File: doc/releases/changelog-dev.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 * `PauliWord` sparse matrices are much faster, which directly improves `PauliSentence`.
   [#4272](https://github.com/PennyLaneAI/pennylane/pull/4272)
 
+* The default label for a `StatePrep` operator is now a ket with psi in it.
```
Comment (mudit2812) [inline doc/releases/changelog-dev.md:12]:
```suggestion
* The default label for a `StatePrep` operator is now a $|\psi >$ in it.
```

**Review by mudit2812 — APPROVED** [review]
Non-blocking suggestion 🚀

**Review by trbromley — APPROVED** [review]
Thanks @timmysilv! Overall I think this is an improvement :+1: Though I do wonder if it makes sense to have the wires be drawn to the left of the `psi` box :thinking: For me it depends on whether we're treating `psi` as an input, or whether the circuit itself is enacting the unitary that prepares `psi` from input `0` states.

File: doc/releases/changelog-dev.md [hunk 2]
```diff
@@ -18,6 +18,9 @@
 * QNode transforms in `qml.qinfo` now support custom wire labels.
   [#4331](https://github.com/PennyLaneAI/pennylane/pull/4331)
 
+* The default label for a `StatePrep` operator is now $|\psi \rangle$.
```
Comment (albi3ro) [inline doc/releases/changelog-dev.md:21]:
```suggestion
* The default label for a `StatePrep` operator is now `|Ψ⟩`.
```

**Review by albi3ro — COMMENTED** [review]
Looks like that angle bracket character is actually fixed width! I remember choosing to use `<` and `>` when refreshing the drawer because that alternative wasn't fixed width, which messed up alignment.  

We should actually consider using this character for expectation values as well.

```
0: ─╭|Ψ⟩──────╭S─┤  <𝓗(M0)>
1: ─╰|Ψ⟩──────├●─┤         
2: ──RX(1.20)─╰●─┤         
```

**Pull request merged**